IBAnimatable/TransitionFromDirection.swift

```diff
@@ -25,6 +27,8 @@ public enum TransitionFromDirection {
     case .Bottom:
       // The actual transition direction is oposite, need to reverse
       return kCATransitionFromTop
+    default:
+      return ""
```

Comment:
@tbaranes I think this property `stringValue` is  only used in `CATransition` and will not fall into `default` at all. Let's leave it `""`. Please merge it when you are ready, thanks.